lredis/cqueues.lua

```diff
@@ -33,6 +33,35 @@ local function connect_tcp(host, port)
 	return new(socket)
 end
 
+local function connect(url)
+	local rest, auth,host,port,db
+	if url then
```

Comment:
Omitting the url to connect to the default server is a common idiom in most redis clients:
```lua
 local lredis = require "lredis.cqueues"
local r = lredis.connect() -- connect to "localhost:6739" by default. 
```